### PR TITLE
Create install size scan tests

### PR DESCRIPTION
**Pull request by darklinkpower**
I have verified that:
- [x] These changes work, by building the application and testing them.
- [x] Pull request is targeting `devel` branch.
- [x] I added myself into [contributors file](https://github.com/JosefNemec/Playnite/blob/devel/source/Playnite.DesktopApp/Resources/contributors.txt) if I want to receive contribution credit in the application itself.

### CODE REVIEW

File: source/Tests/Playnite.Tests/FileSystemTests.cs [hunk 1]
```diff
@@ -68,5 +68,27 @@ public void CheckSumTests()
             StringAssert.AreEqualIgnoringCase("46fcb37aa8e69b4ead0d702fd459299d", FileSystem.GetMD5(testFile));
             StringAssert.AreEqualIgnoringCase("D8B22F5D", FileSystem.GetCRC32(testFile));
         }
+
+        [Test]
+        public void DirectorySizeScanTest()
+        {
+            var testDir = Path.Combine(PlayniteTests.TempPath, "TestDir");
```
Comment (JosefNemec) [inline source/Tests/Playnite.Tests/FileSystemTests.cs:75]: You can use `using (var tempPath = TempDirectory.Create())` to create temp test directory that will be empty and specific for your test, so you don't have to make sure you have any files cleared from previous tests etc. See `ScannerTests` for how to use it.

Comment (darklinkpower) [reply source/Tests/Playnite.Tests/FileSystemTests.cs]: Done in https://github.com/JosefNemec/Playnite/pull/3059/commits/a00847bdedb4c7d97f941671f6b3069d8a14698f

File: source/Tests/Playnite.Tests/FileSystemTests.cs [hunk 2]
```diff
@@ -68,5 +68,28 @@ public void CheckSumTests()
...
             StringAssert.AreEqualIgnoringCase("D8B22F5D", FileSystem.GetCRC32(testFile));
         }
+
+        [Test]
+        public void DirectorySizeScanTest()
+        {
+            using (var tempPath = TempDirectory.Create())
+            {
+                var filePath = Path.Combine(tempPath.TempPath, "DummyFile");
+                FileSystem.DeleteFile(filePath);
```
Comment (JosefNemec) [inline source/Tests/Playnite.Tests/FileSystemTests.cs:78]: Not needed since TempDirectory is created empty by default.

Comment (darklinkpower) [reply source/Tests/Playnite.Tests/FileSystemTests.cs]: Fixed in https://github.com/JosefNemec/Playnite/pull/3059/commits/8c18d0578183974b3ffd9339f33e9b4a167cc511

**Pull request merged**